# Messages
int32 center_x
int32 center_y
string robot_state
string action_state
int32  pan_position
int32  tilt_position
string follow_ball_status  
int32  standing_status
string kick_status
string detection_state
int32 x_line
int32 y_line
